dist/index.js

```diff
@@ -24,11 +24,23 @@ var getBlogPosts = async function(lang) {
         out += "description: " + post.Description + "\n";
         out += "author: '" + post.AuthorAndTitle + "'\n";
         out += "date: '" + post.PublishDate + "'\n";
-        out += "image: " + post.BannerImage.url + "\n";
+        out += "image: " + post.BannerImage.url.replace(
```

Comment:
So, just so I understand, the index.js file is run during a build step. This will substitute all of the images we have in content files with the cloudfront URL. In theory, this will become redundant in the future as we'll never put s3 URLs in to the docs, they'll end up as cloudfront URLs directly?

Reply:
No, unfortunately the CMS can only deal with the s3 URLs directly, unless we want to fork and maintain the provider code, so the data coming from the CMS will always have the s3 url, and need to be converted here.

Reply:
Ah I see, and I'm guessing we run this and generate a static site? It's done on our side, not per client, yea?

Would it be worth it to add a comment that this fixes up a mis-match between the software and our deploy by converting the s3 url that is always set by the CMS to the cloudfront url?

Reply:
Probably yes 👍 Will do

Reply:
And yes, the site is static, and generated downstream in digital-canada-ca after merging to master